go/ct/spc/specification.go

```diff
@@ -2362,3 +2397,11 @@ func sumWithOverflow(values ...vm.Gas) (vm.Gas, bool) {
 	}
 	return res, false
 }
+
+func wordCountWithOverflow(size uint64) (vm.Gas, bool) {
```

Comment:
The result should not be of the type `vm.Gas` since it is a number of words.

Reply:
correct